eng/stryker-config.json

```diff
@@ -19,6 +19,7 @@
       "block",
       "statement"
     ],
+    "concurrency": 4,
```

Comment:
Based on [`concurrency`](https://stryker-mutator.io/docs/stryker-js/configuration/#concurrency-number) and [Standard GitHub-hosted runners for public repositories](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories), looks like the default would have been `4` anyway?

> Set the concurrency of workers. This defaults to `n-1` where `n` is the number of logical CPU cores available on your machine, unless `n <= 4`, in that case it uses `n`.

Reply:
I think [this](https://github.com/stryker-mutator/stryker-net/blob/b873b6268c7bdd9ca7529ac50e6329d05327f389/src/Stryker.Options/Options/Inputs/ConcurrencyInput.cs#L18) solves the mystery.

```csharp
public override int? Default => Math.Max(Environment.ProcessorCount / 2, 1);
```

The documentation is wrong, so it's actually using a concurrency of `2` by default.

Reply:
Nope, turns out _actually_ that was the documentation for stryker-js. 🤦

[The docs for .NET are correct](https://github.com/stryker-mutator/stryker-net/blob/b873b6268c7bdd9ca7529ac50e6329d05327f389/docs/configuration.md?plain=1#L303), they're just not what Google found as the top result when I searched for the Stryker docs.